module/kvmfr.c

```diff
@@ -351,6 +396,86 @@ static struct pci_driver kvmfr_pci_driver =
   .remove   = kvmfr_pci_remove
 };
 
+// Must be run with minor_lock
+static int create_static_device(int size_mb)
```

Comment:
The standard convention is to name a method that requires locking as `_unlocked` as such:
`create_static_device_unlocked`

Reply:
Done.